Index.html

```diff
@@ -26,244 +27,263 @@
+    <p class="intro">
+      Hello I’m a software developer! I can help you build a product ,
+      feature or website. Look through some of my work and experience! If
+      you like what you see and have a project you need coded, don’t
+      hesitate to contact me.
+    </p>
+    <h3 class="Connect">LET'S CONNECT</h3>
+    <ul class="social-1">
+      <li>
+        <a
+          class="linkedin"
+          href="https://www.linkedin.com/in/kerlos-bahaa-457a8925a/"
+        >
+          <i class="fa-brands fa-linkedin-in"></i>
+        </a>
+      </li>
+      <li>
+        <a href="#"> <img src="./picture/Vector.png" alt="" /></a>
+        
+      </li>
+      <li>
+        <a class="github" href="https://github.com/kerlos-bahaa">
+          <i class="fa-brands fa-github"></i>
+        </a>
+      </li>
+      
+      <li>
+        <a href="#" > <img src="./picture/Vector(1).png" alt="" /></a>
+        
+      </li>
+      <li>
+        <a class="twitter" href="https://twitter.com/KerlosBahaa"
+          ><i class="fa-brands fa-twitter"></i>
+        </a>
+      </li>
+    </ul>
+    <button class="resume">
+      Get my resume </button>
 
-          <li>
-            <a href="#"> <img src="./picture/Vector(1).png" alt="" /></a>
-          </li>
-          <li>
-            <a class="twitter" href="https://twitter.com/KerlosBahaa"
-              ><i class="fa-brands fa-twitter"></i>
-            </a>
-          </li>
-        </ul>
-        <button class="resume">Get my resume</button>
-      </div>
-      <div>
+  </div>
+      <div class="aboutmyself-bottom">
         <ul class="end-part">
-          <li class="end-part-1">Languages</li>
+          <li class="end-part-1">
+            Languages&nbsp;
+          </li>
           <ul class="skil">
-            <li>
-              <img src="./picture/Ellipse 1.png" alt="" />
-              <p>JavaScript</p>
-            </li>
-            <li>
-              <img src="./picture/Ellipse 2.png" alt="" />
-              <p>HTML</p>
-            </li>
-            <li>
-              <img src="./picture/Ellipse 1(1).png" alt="" />
-              <p>CSS</p>
-            </li>
+            <li> <img class="image-java" src="./picture/Ellipse 1.png" alt=""> <p>JavaScript</p></li>
+            <li><img src="./picture/Ellipse 2.png" alt=""> <p>HTML</p></li>
+            <li><img src="./picture/Ellipse 1(1).png" alt=""> <p>CSS </p></li>
           </ul>
           <li class="end-part-2">Frameworks</li>
-          <hr />
-          <li class="end-part-3">Skills</li>
-          <hr />
+          <hr>
+          <li class="end-part-3">Skills&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li>
+          <hr>
         </ul>
       </div>
     </section>
-
     <footer>
       <section class="footer-section">
         <form
           action="https://formspree.io/f/xayzpwaq"
           method="post"
           target="_blank"
         >
+        <!-- contact me  -->
```

Comment:
- [ ] please kindly use the correct background image in the desktop contact section like in the original design.
yours
![Screenshot (913)](https://user-images.githubusercontent.com/65298048/228685266-254b50ed-a951-41cc-ac4e-0de5569fbbb5.png)
figma
![Screenshot (914)](https://user-images.githubusercontent.com/65298048/228685289-a2f71027-aaa8-4326-b006-9a64f3193555.png)